org.gnome.Hammond.json

```diff
@@ -0,0 +1,37 @@
+{
+    "app-id": "org.gnome.Hammond",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "hammond",
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "build-options" : {
+        "env": {
+            "CARGO_HOME": "/run/build/Hammond/cargo"
+        },
+        "build-args": [ "--share=network" ]
```

Comment:
You can't have network access, you need to do releases with all dependencies bundled (`cargo lock` I believe?).

Reply:
Oh yea, good thing that they already included in the tarball. 
It builds fine without network access. Pushing a fix in a bit.